src/mjolnir/osmway.cc

```diff
@@ -1069,7 +1069,33 @@ void OSMWay::GetTaggedValues(const UniqueNames& name_offset_map,
     // level
     auto tokens = GetTagTokens(name_offset_map.name(level_index_));
     for (const auto& t : tokens) {
-      names.emplace_back(encode_tag(TaggedValue::kLevel) + t);
+
```

Comment:
This is the actual parsing of the initial string: 
- splits it by semicolon
- for each token, check if the token denotes a range
- if it is a range, add a tagged value for each value in the range
- if it is not a range, simply store the value

Another way would be to push this out to edgeinfo and just store the string as a single tagged value, but I suppose this is sparse data (and most edges only connect 2 floors anyways).

Reply:
Maybe I'm mistaken but shouldn't the largest two byte varint be `(2**13)-1=8191`? 6 bits on the first (one for the sign), 7 on the second byte

Reply:
yep you have it right. when i was just talking off the cuff earlier today and doing the math in my head i made two mistakes in quick succession:

1. i didnt think about the sign, which made me think it was 7 + 7 = 14 bits
2. i misremembered that 2^16 isnt 16k but rather 64k, so i took 16k in half 2 times rather than 64k in half 2 times